src/avatar.scss

```diff
@@ -0,0 +1,153 @@
+@import "./mixins";
+
+/*!
+.fd-avatar+((--circle), (--thumbnail), (--xxs|--xs|--s|--m|--l|--xl|--xxl))
+*/
+$block: #{$fd-namespace}-avatar;
+
+.#{$block} {
+  // LOCAL VARS (set all themeable properties, always include !default)
+  $fd-const-ratio: 1rem;
+  $fd-avatar-sizes: (
+    xs: $fd-const-ratio * 2,
+    s: $fd-const-ratio * 3,
+    m: $fd-const-ratio * 4,
+    l: $fd-const-ratio * 5,
+    xl: $fd-const-ratio * 7,
+  ) !default;
+  $fd-avatar-border-radius: $fd-border-radius !default;
```

Comment:
I wouldn't set this value to the variable coming form the _settings. It might be used in other places and if someone change it for their purposes it will reflect here. It's better to set a local var (at least in my opinion)